bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/widgets/Table.java

```diff
@@ -132,6 +132,9 @@ public class Table extends Composite {
  */
 public Table (Composite parent, int style) {
 	super (parent, checkStyle (style));
+
+	this.nativeItemHeight = (int)((NSTableView)view).rowHeight();
```

Comment:
This returns 24 whereas the calculated height with font metrics as returned by `getItemHeight` is 25. I'm not sure whether that matters?

Reply:
It may return an arbitrary value different from 24 (especially if smallFonts is set or when you're on an older macOS). We only use that value as a minimum. In the end, it seems to work out.